sdk/go/common/workspace/plugins.go

```diff
@@ -423,7 +423,7 @@ func (info PluginInfo) installLock() (unlock func(), err error) {
 // If a failure occurs during installation, the `.partial` file will remain, indicating the plugin wasn't fully
 // installed. The next time the plugin is installed, the old installation directory will be removed and replaced with
 // a fresh install.
-func (info PluginInfo) Install(tgz io.ReadCloser) error {
+func (info PluginInfo) Install(tgz io.ReadCloser, reinstall bool) error {
```

Comment:
Just curious - this is not a change we'd consider breaking? Noting it's in sdk/go and it's a public method that changes signature, technically. Thinking it through though I can't imagine there's user code that calls it.

Reply:
Last I asked @mikhailshilkov about this the comment was we should probably be keeping sdk/go/common compatible.

Reply:
I can't imagine that there's user code that uses this either. Though this is technically breaking, I personally am not concerned about it. If we want to change it, I'd propose two exported methods: `Install` with its current signature and `Reinstall` (also with the same signature).